### PR TITLE
Add 6 new companies to the list

### PR DESCRIPTION
**Pull request by madhusudhan1234**
- [x] Heartsun Technology
- [x] Helloworld Corp
- [x] Microsoft Innovation Center Nepal
- [x] Unilimited Technology Pvt. Ltd.
- [x] Websoft Nepal Pvt. Ltd.
- [x] UBA Solutions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,9 +133,11 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
...
 - [ThemeGrill](https://themegrill.com/) | Basundhara, Kathmandu | Free and Premium WordPress Themes Store
 - [Three Monks](http://www.threemonk.com/) | New Baneshwor, Kathmandu | Carter the growing needs in the space of technical expertise with special focus on Digital Financial Services, Money Transfer Business and Entertainment Companies
+- [UBA Solutions](http://www.uba-solutions.com/) | Thadodhunga, Lalitpur | Software Research and Development
 - [Unelma Platforms](http://unelmaplatforms.com/) | Ratnanagar, Chitwan | Unelma Platforms is a provider of software development services for higher-education, healthcare and business markets
 - [UniAxilSoftwares](http://uniaxialsoftwares.com/) | Anamnagar, Kathmandu | Innovation - for real world problems
 - [Unifun](https://unifun.com/) | Nagpokhari, Kathmandu | Unifun is a provider of high-class IT products for Mobile Operators
+- [Unlimited Technology Pvt Ltd](http://unlimit.com) | Unlimited Building, Khichapokhari | expertise in business software, custom software, business intelligence, document management/collaboration, mobile and infrastructure solutions
```
Comment (mesaugat) [inline README.md]: Can you make this `Khichapokhari, Kathmandu`. 

Also, can you start the description with a capital letter. It should be:

```diff
- expertise
+ Expertise
```

File: README.md [hunk 2]
```diff
@@ -141,6 +146,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Vianet Communications Pvt. Ltd.](https://www.vianet.com.np/) | Jawalakhel, Lalitpur | Provider of high speed Internet connectivity and communications solutions for business and individuals
 - [Visualise VR](http://visualisevr.com/) | Pulchowk, Lalitpur |  Help business benefit from mixed reality technologies
 - [Webpage Nepal Pvt. Ltd.](http://webpagenepal.com/) | New Road, Pokhara | Pokhara based web development company
+- [WebSoft Nepal Pvt. Ltd.](http://www.websoftnepal.com) | Pashupati Plaza, Indradev Marg, Narayangarh | Provides a complete solution on Web Design and Hostings
```
Comment (mesaugat) [inline README.md]: `Pashupati Plaza, Indradev Marg, Narayangarh`

Can you shorten this down to `Intradev Marg, Narayangarh` or `Pashupati Plaza, Narayangarh`?

File: README.md (hunk 1 above)
Comment (mesaugat) [inline README.md]: Also, `Pvt. Ltd.`

File: README.md [hunk 3]
```diff
@@ -100,6 +102,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Mantra Ideas Pvt. Ltd.](http://mantraideas.com/) | Sankhamul , Kathmandu | IT Company providing services related to web and mobile app development
 - [Mercantile Communications Pvt. Ltd.](http://www.mos.com.np/) | Durbar Marg, Kathmandu | Provides comprehensive range of excellent value added services for Internet, content designing, development and hosting
 - [Mhuchya Technology](https://mhuchya.com.np/) | Sankhamul, Kathmandu | United for the development of Technology in Nepal, globally
+- [Microsoft Innovation Center Nepal](http://micnepal.org) | Unlimited Building, Khichapokhari | provides access to world-class resources for students, software developers, IT professional& entrepreneurs
```
Comment (mesaugat) [inline README.md]: Let's make it `Khichapokhari, Kathmandu`.

File: README.md [hunk 4]
```diff
@@ -73,6 +73,8 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [GrowByData](http://growbydata.com/) | Kupondole, Lalitpur | Ecommerce data services for online retailers
 - [Growcept](https://growcept.com/) | Pulchowk, Lalitpur | Growcept is a tech products platform
 - [Gurkhatech](https://gurkhatech.com/) | New Baneshwor, Kathmandu | App & Web Development, Facebook Marketing Nepal
+- [Heartsun Technolpogy](http://heartsunntsoft.com/) |  Bharatpur-10, Chitwan|Software Company with DotNet platform
```
Comment (mesaugat) [inline README.md]: Format is not correct:

```diff
- |  Bharatpur-10, Chitwan|Software Company with DotNet platform
- | Bharatpur, Chitwan | Software Company working with .NET platform
```

Comment (mesaugat) [reply README.md]: Typo
```diff
- Heartsun Technolpogy
+ Heartsun Technology
```

**Comment by madhusudhan1234** [comment]
Hi @mesaugat , Thank You for the review. I made changes according to your PR review suggestion.

**Pull request merged**